includes/data-stores/class-wc-admin-reports-taxes-data-store.php

```diff
@@ -296,7 +296,7 @@ protected function normalize_order_by( $order_by ) {
 		global $wpdb;
 
 		if ( 'rate' === $order_by ) {
-			return $wpdb->prefix . 'woocommerce_tax_rates.tax_rate';
+			return "CAST({$wpdb->prefix}woocommerce_tax_rates.tax_rate as DECIMAL(18,4))";
```

Comment:
I imagine `100.0000` is the upper limit on taxes, should this be `DECIMAL(7,4))`? Or `DECIMAL(5,2))` for two decimal points.

Reply:
Makes sense! I investigated it a bit and the upper limit seems to be `999.9999`. That's because values are stored in a `varchar(8)` and the UI doesn't impose any other limit than that, so percentages over 100% are allowed.

Fixed in 465b9b1fc0efae8547c52050057e483098dc16c0.

> Or DECIMAL(5,2)) for two decimal points.

I don't think there is any big performance difference between both, is it? So probably `DECIMAL(7,4)` is a better option, so sorting is more precise.